app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -2759,11 +2761,13 @@ class BrowserTabViewModel @Inject constructor(
             val isBrowserShowing = currentBrowserViewState().browserShowing
             val isErrorShowing = currentBrowserViewState().maliciousSiteBlocked
             if (hasCtaBeenShownForCurrentPage.get() && isBrowserShowing) return null
+            val showBrokenSitePrompt = shouldShowPromptForBrokenSiteReport()
             val cta = withContext(dispatchers.io()) {
                 ctaViewModel.refreshCta(
                     dispatchers.io(),
                     isBrowserShowing && !isErrorShowing,
                     siteLiveData.value,
+                    showBrokenSitePrompt,
```

Comment:
Overall I'm struggling with this. I don't fully get why we have pull this out of the `CtaViewModel` to pass a boolean back to it when within the `CtaViewModel` we still have access to the `BrokenSitePrompt`. It seems an unnecessary journey for a value that we can directly get when needed in the `CtaViewModel`